### ISSUE TITLE
bug: Unable to answer any review items (answer always responds with text retry jiggle)

### ISSUE THREAD

**Issue by wmgroot** [davidsansome/tsurukame#603]
This started occurring for me at least a month ago, but I took a break hoping that the issue would be resolved. But it's still happening for me now.

## Info
iPhone 11
iOS Version: 16.1.2
Tsurukame Version: 1.25.925

## Bug Description
After successfully reviewing a few items, I enter a state where no matter what answer I enter, the app responds with the text retry jiggle that is normally provided when you give the onyomi reading instead of kunyomi (or vice versa). This blocks me from performing any review at all.

## Steps I've taken to try to resolve this issue.
1. Verified that I do have an active WaniKani subscription.
2. Reinstall Tsurukame, signed in fresh with my active account.
3. Close the Tsurukame app and re-open. Issue is still present immediately after rebooting the app.
4. End review session. When starting a new session, I can successfully complete 1-2 items before the issue returns.
5. Refreshing by pulling down on the home screen appears to have no impact on this issue

## Example Review
Prompt: 兵
Answer: へい
Result: Success

Prompt: 兵
Answer: へい
Result: Text jiggles, cannot move on
Answer: さ
Result: Text jiggles, cannot move on
... etc

**Comment by UInt2048** [comment]
Are you answering the reading when you’re supposed to be answering the meaning in the second case? Tsurukame prohibits you from typing Japanese in the meaning.iPhoneから送信2022/12/16 13:17、wmgroot ***@***.***>のメール:﻿
This started occurring for me at least a month ago, but I took a break hoping that the issue would be resolved. But it's still happening for me now.
Info
iPhone 11
iOS Version: 16.1.2
Tsurukame Version: 1.25.925
Bug Description
After successfully reviewing a few items, I enter a state where no matter what answer I enter, the app responds with the text retry jiggle that is normally provided when you give the onyomi reading instead of kunyomi (or vice versa). This blocks me from performing any review at all.
Steps I've taken to try to resolve this issue.

Verified that I do have an active WaniKani subscription.
Reinstall Tsurukame, signed in fresh with my active account.
Close the Tsurukame app and re-open. Issue is still present immediately after rebooting the app.
End review session. When starting a new session, I can successfully complete 1-2 items before the issue returns.
Refreshing by pulling down on the home screen appears to have no impact on this issue

Example Review
Prompt: 兵
Answer: へい
Result: Success
Prompt: 兵
Answer: へい
Result: Text jiggles, cannot move on
Answer: さ
Result: Text jiggles, cannot move on
... etc
